index.html

```diff
@@ -36,5 +36,144 @@ <h6 class="footer">LET'S CONNECT</h6>
+                    </div>
+                    <div><button class="button-big">See Project</button></div>
+                </div>
+                
+                <div class="grid-item">
+                    <div class="work-image"><img src="./Images/Works/SnapshootPortfolio4.png" alt="work image"></div>
+                    <h2 class="headline-work">Multi-Post Stories</h2>   
+                    <ul class="headline-detail">
+                        <li class="canopy">CANOPY</li>
+                        <li><img src="./Images/Works/Counter.png" alt="counter"></li>
+                        <li class="backenddev">Back End Dev</li> 
+                        <li><img src="./Images/Works/Counter.png" alt="counter"></li>
+                        <li class="backenddev">2015</li>                                
+                    </ul>        
+                    <p class="headlinecont-work">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div>                                
+                        <ul class="smallbuttons">
+                            <li><button class="button-small">html</button></li>
+                            <li><button class="button-small">css</button></li>
+                            <li><button class="button-small">JavaScript</button></li>
+                        </ul>
+                    </div>
+                    <div><button class="button-big">See Project</button></div>
+                </div>
+            </div>
+
+        </section>
+    </div>
+    
+        <section>
+            <div class="grid-container2">
+                <div class="grid-item2">
+                    <h1 class="final-headline">About <br>Myself</h1>
+                    <p class="final-cont">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div>
+                    <h6 class="connect">LET'S CONNECT</h6>
+                    <ul class="socialicons2">
+                        <li><a class="image 1" href="https://www.twitter.com/home"><img src="./Images/Socialmedia/Vector.png" alt="twitter icon"></a></li>
+                        <li><a class="image 2" href="https://www.linkedin.com/in/alejandra-keber"><img src="./Images/Socialmedia/Linkedinicon.jpg" alt="likedin icon"></a></li>
+                        <li><a class="image 3" href="https://www.microverse.org/"><img src="./Images/Socialmedia/Vector(1).png" alt="microverse icon"></a></li>
+                        <li><a class="image 4" href="https://www.github.com/AlejandraKeber"><img src="./Images/Socialmedia/Vector(2).png" alt="github icon"></a></li>
+                        <li><a class="image 5" href="https://www.google.com/"><img src="./Images/Socialmedia/Vector(3).png" alt="vector3 icon"></a></li>
```

Comment:
Kindly. class names should not have spaces (unless they are 2 different classes).